invenio_records_rest/config.py

```diff
@@ -58,20 +56,159 @@ def _(x):
         max_result_window=10000,
     ),
 )
+"""Default REST endpoints loaded.
+
+This option can be overwritten to describe the endpoints of the different
```

Comment:
IMHO the second article should not be there (consider: `the endpoints of different record types.` as there is no specific group of record types).